.github/dependabot.yml

```diff
@@ -48,3 +48,12 @@ updates:
       timezone: "Europe/Amsterdam"
     reviewers:
     - "infonl/dimpact"
+
+  - package-ecosystem: "docker"
```

Comment:
Our Dockerfile is named 'Containerfile'. Does it pick this up? I understood from @bas-info-nl you need to configure this.

Reply:
`Creating a PR to see if Bas already fixed this, no need to review yet`